yarn.lock

```diff
@@ -116,6 +144,17 @@
     levenary "^1.1.1"
     semver "^5.5.0"
 
+"@babel/helper-compilation-targets@^7.8.7":
```

Comment:
Maybe we can do something to deduplicate these changes? `npx yarn-deduplicate yarn.lock`?